setup.py

```diff
@@ -29,7 +29,7 @@ def run_tests(self):
     'coverage>=4.5, <5',
     'pylint>=1.9, <2',
     'pytest>=4, <5',
-    'pytest-cov>=2, <3',
+    'pytest-cov==2.7.1',
```

Comment:
is there a reason this needs to be pinned at 2.7.1 instead of, e.g. between 2 and 2.7.1?